core/services/workflows/v2/engine.go

```diff
@@ -488,17 +488,13 @@ func (e *Engine) deductStandardBalances(meteringReport *metering.Report) {
 	// Add an extra second of metering padding for context cancel propagation
 	ctxCancelPadding := (time.Millisecond * 1000).Milliseconds()
 	compMs := decimal.NewFromInt(int64(e.cfg.LocalLimits.WorkflowExecutionTimeoutMs) + ctxCancelPadding)
-	computeUnit := billing.ResourceType_name[int32(billing.ResourceType_RESOURCE_TYPE_COMPUTE)]
-	computeAmount, mrErr := meteringReport.ConvertToBalance(computeUnit, compMs)
-	if mrErr != nil {
-		e.lggr.Errorw("could not determine compute amount to meter", "err", mrErr)
-	}
+	computeUnit := billing.ResourceType_RESOURCE_TYPE_COMPUTE.String()
 
-	if mrErr = meteringReport.Deduct(
+	if _, err := meteringReport.Deduct(
 		computeUnit,
-		computeAmount,
-	); mrErr != nil {
-		e.lggr.Errorw("could not meter compute", "err", mrErr)
+		metering.WithNativeValue(computeUnit, compMs),
+	); err != nil {
+		e.lggr.Errorw("could not deduct balance for capability request", "capReq", "standard-deduction", "err", err)
```

Comment:
```suggestion
		e.lggr.Errorw("could not deduct balance for capability request", "capReq", "standard-deduction-compute", "err", err)
```

I can envision this being not clear at a glance if parsing error logs. Adding the compute part might help.